clients/src/test/java/org/apache/kafka/clients/NetworkClientTest.java

```diff
@@ -62,10 +62,12 @@
     protected final long reconnectBackoffMaxMsTest = 10 * 10000;
 
     private final TestMetadataUpdater metadataUpdater = new TestMetadataUpdater(Collections.singletonList(node));
+    private final TestClusterMetadataUpdater clusterMetadataUpdater = new TestClusterMetadataUpdater(Collections.singletonList(node));
```

Comment:
Is it possible to have integration tests to exercise code in the patch?  e.g., setting cluster metadata expiration to a small number

Reply:
added another test in MetadataTest, cannot simulate the exact behavior by calling NetworkClient methods since the metadataUpdater is private.